packages/dev/loaders/src/glTF/2.0/Extensions/EXT_materials_diffuse_roughness.ts

```diff
@@ -0,0 +1,102 @@
+import type { Nullable } from "core/types";
+import { PBRMaterial } from "core/Materials/PBR/pbrMaterial";
+import type { Material } from "core/Materials/material";
+
+import type { IMaterial } from "../glTFLoaderInterfaces";
+import type { IGLTFLoaderExtension } from "../glTFLoaderExtension";
+import { GLTFLoader } from "../glTFLoader";
+import type { IKHRMaterialsDiffuseRoughness } from "babylonjs-gltf2interface";
+import { registerGLTFExtension, unregisterGLTFExtension } from "../glTFLoaderExtensionRegistry";
+import { Constants } from "core/Engines/constants";
+
+const NAME = "EXT_materials_diffuse_roughness";
+
+declare module "../../glTFFileLoader" {
+    // eslint-disable-next-line jsdoc/require-jsdoc
+    export interface GLTFLoaderExtensionOptions {
+        /**
+         * Defines options for the EXT_materials_diffuse_roughness extension.
+         */
+        // NOTE: Don't use NAME here as it will break the UMD type declarations.
+        ["EXT_materials_diffuse_roughness"]: {};
+    }
+}
+
+/**
+ * [Specification](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/EXT_materials_diffuse_roughness/README.md)
+ * [Playground Sample](https://www.babylonjs-playground.com/frame.html#7F7PN6#8)
+ */

```

Comment:
Link is dead. Playground sample is for something else. Spec is PR, so we should mark as experimental.
```suggestion
/**
 * [Specification](https://github.com/KhronosGroup/glTF/blob/fdee35425ae560ea378092e38977216d63a094ec/extensions/2.0/Vendor/EXT_materials_diffuse_roughness/README.md)
 * @experimental
 */

```